services/web/server/src/simcore_service_webserver/api/v0/openapi.yaml

```diff
@@ -203,7 +203,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/paths/~1auth~1login/post/responses/200/content/application~1json/schema'
+                $ref: '#/paths/~1auth~1login/post/responses/202/content/application~1json/schema'
```

Comment:
is this happening again?  remember? the openapi composer would take different equivalent paths depending of the version

Reply:
shit, yes. I need to double check that thing again